examples/remote_procedure/mnist/webapp/cypress/e2e/remote_procedure_mnist.cy.js

```diff
@@ -0,0 +1,87 @@
+describe("Loads remote_procedure_mnist", () => {
+  it("Makes request for agent", () => {
+    cy.intercept({ pathname: "/request_agent" }).as("agentRequest");
+    cy.visit("/");
+    cy.wait("@agentRequest").then((interception) => {
+      expect(interception.response.statusCode).to.eq(200);
+    });
+  });
+  it("Loads correct react elements", () => {
+    cy.visit("/");
+    cy.get('[data-cy="canvas-container-0"]');
+    cy.get('[data-cy="clear-button-0"]');
+    cy.get('[data-cy="correct-checkbox-0"]');
+    cy.get('[data-cy="correct-text-input-0"]');
+
+    cy.get('[data-cy="canvas-container-1"]');
+    cy.get('[data-cy="clear-button-1"]');
+    cy.get('[data-cy="correct-checkbox-1"]');
+    cy.get('[data-cy="correct-text-input-1"]');
+
+    cy.get('[data-cy="canvas-container-2"]');
+    cy.get('[data-cy="clear-button-2"]');
+    cy.get('[data-cy="correct-checkbox-2"]');
+    cy.get('[data-cy="correct-text-input-2"]');
+
+    cy.get('[data-cy="submit-button"]').as("submitButton");
+    cy.get("@submitButton").should("be.disabled");
+  });
+
+  it("Submitting with three corrected annotations", () => {
+    cy.visit("/");
+    cy.on("window:alert", (txt) => {
+      expect(txt).to.contain("The task has been submitted!");
+    });
+
+    cy.get('[data-cy="clear-button-0"]').as("clearButton0");
+    cy.get('[data-cy="clear-button-1"]').as("clearButton1");
+    cy.get('[data-cy="clear-button-2"]').as("clearButton2");
+
+    // draw 4
+    cy.get('[data-cy="canvas-mouse-down-container-0"]')
+      .trigger("mouseover")
+      .trigger("mousedown", 20, 20)
+      .trigger("mousemove", 40, 150)
+      .trigger("mousemove", 150, 120)
+      .trigger("mousemove", 150, 10)
+      .trigger("mouseup", 150, 220);
+
+    // There is a wait statement here because it takes some time for the model to calculate that it is a 4
+    cy.wait(2000);
+    cy.get('[data-cy="current-annotation-0"]').should("contain.text", "4");
+    cy.get('[data-cy="correct-checkbox-0"]').check();
+    cy.get('[data-cy="correct-text-input-0"]').should("not.exist");
+
+    // draw 1
+    cy.get('[data-cy="canvas-mouse-down-container-1"]')
+      .trigger("mouseover")
+      .trigger("mousedown", 100, 20)
+      .trigger("mousedown", 120, 200)
+      .trigger("mouseup", 120, 200);
```

Comment:
This draws a 1, but in the GitHub action it seems to sometimes think it is a 8 leading to the test failing about 1 out of every 4 times. Might make more sense to draw a number that is clearer to the model.

Reply:
It feels kinda weird though because given the exact same mouse down values I feel like the model should give me the exact same result, but I guess that is not the case. AI stuff I guess 🤷

Reply:
Changed the 1 to a 3, hopefully this is more consistent